projects/demo/src/modules/app/stackblitz/project-files/configs.md

```diff
@@ -79,5 +79,3 @@
   }
 }
 ```
-
-Ivy is disabled because of Stackblitz error: https://github.com/stackblitz/core/issues/1364
```

Comment:
It is not true after this PR https://github.com/Tinkoff/taiga-ui/pull/2076/files